question_three.py

```diff
@@ -0,0 +1,23 @@
+import operator as op
+
+sentences = ['My name is Ram', 'He is a good person', 
+             'You should be careful while coding', 'He can do better', 
+             'The person is mysterious', 'Jay Shree Ram', 'It is my pen']
+word_trees=[]
+for i in sentences:
+    res=i.split(' ')
+    word_trees.append(res)
+
+print('\nword_trees = {}'.format(word_trees))
+print('\nNumber of time each word appears : ')
+occurrence={}
+sentences=[]
+for i in word_trees:
+    sentences+=i
+    
+for i in sentences:
+        res = op.countOf(sentences,i)
+        occurrence[i]=res
```

Comment:
One tab is extra for the indentation. Please correct the indentation. The indentation in python is too important, so make sure we give the correct indentation.